website/content/docs/platform/k8s/vso/sources/vault/auth/index.mdx

```diff
@@ -0,0 +1,501 @@
+    "logger": "cachingClientFactory",
+    "msg": "Failed to get cacheKey from obj",
+    "controller": "vaultstaticsecret",
+    "controllerGroup": "secrets.hashicorp.com",
+    "controllerKind": "VaultStaticSecret",
+    "VaultStaticSecret": {
+      "name": "app1",
+      "namespace": "apps"
+    },
+    "namespace": "apps",
+    "name": "app1",
+    "reconcileID": "5201f597-6c5d-4d07-ae8f-30a39c80dc54",
+    "error": "failed getting admin/default, err=VaultAuthGlobal.secrets.hashicorp.com \"default\" not found"
+  }
+  ```
+
+  Check for related Kubernetes events:
+
+  ```shell
+  $ kubectl events --types=Warning -n admin --for vaultauths.secrets.hashicorp.com/default -o json
+  ```
+
+  Sample out from the Kubernetes event for the VaultAuth resource:
+
+  ```json
+  {
+    "kind": "Event",
+    "apiVersion": "v1",
+    "metadata": {
+      "name": "default.17e2c0da7b0e36b5",
+      "namespace": "admin",
+      "uid": "3ca6088e-7391-4b76-9443-a790ccae02c0",
+      "resourceVersion": "634396",
+      "creationTimestamp": "2024-07-16T17:14:12Z"
+    },
+    "involvedObject": {
+      "kind": "VaultAuth",
+      "namespace": "admin",
+      "name": "default",
+      "uid": "1dabe3a5-5479-4f5d-ac48-5db7eff7f822",
+      "apiVersion": "secrets.hashicorp.com/v1beta1",
+      "resourceVersion": "631994"
+    },
+    "reason": "Accepted",
+    "message": "Failed to handle VaultAuth resource request: err=failed getting admin/default, err=VaultAuthGlobal.secrets.hashicorp.com \"default\" not found",
+    "source": {
+      "component": "VaultAuth"
+    },
+    "firstTimestamp": "2024-07-16T17:14:12Z",
+    "lastTimestamp": "2024-07-16T17:15:53Z",
+    "count": 25,
+    "type": "Warning",
+    "eventTime": null,
+    "reportingComponent": "VaultAuth",
+    "reportingInstance": ""
+  }
+  ```
+
+Check the conditions on the VaultAuth resource:
+
+  ```shell
+  $ kubectl get vaultauths.secrets.hashicorp.com -n admin default -o jsonpath='{.status}'
+  ```
+
+Sample output of the VaultAuth's status (prettified). The `valid` field will be `false` for the condition reason
+`VaultAuthGlobalRef`:
+  ```json
+  {
+    "conditions": [
+      {
+        "lastTransitionTime": "2024-07-16T15:35:43Z",
+        "message": "failed getting admin/default, err=VaultAuthGlobal.secrets.hashicorp.com \"default\" not found",
+        "observedGeneration": 3,
+        "reason": "VaultAuthGlobalRef",
+        "status": "False",
+        "type": "Available"
+      }
+    ],
+    "specHash": "e264f241cb4ad776802924b6ad2aa272b11cffd570382605d1c2ddbdfd661ad3",
+    "valid": false
+  }
+  ```
+- Situation: The VaultAuthGlobal resource is not found or is invalid for some reason, denoted by error messages like
+`not found...`.
+
+  Resolution: Ensure that the VaultAuthGlobal resource exists in the namespace specified in the VaultAuth
+resource's or a default VaultAuthGlobal resource exists per
+[VaultAuthGlobal configuration inheritance](#vaultauthglobal-configuration-inheritance)
```

Comment:
I'm not sure what `VaultAuth resource's` is referring to, but maybe something like this?

```suggestion
  Resolution: Ensure that the VaultAuthGlobal resource exists in the referring VaultAuth's namespace or a default
VaultAuthGlobal resource exists per [VaultAuthGlobal configuration inheritance](#vaultauthglobal-configuration-inheritance)
```